datadog_operator/metadata.csv

```diff
@@ -1,2 +1,69 @@
+datadog.operator.go_memstats_last_gc_time_seconds,gauge,,,,Number of seconds since 1970 of last garbage collection,0,datadog_operator,go_memstats_last_gc_time_seconds,
+datadog.operator.go_memstats_lookups_total,count,,,,Total number of pointer lookups,0,datadog_operator,go_memstats_lookups_total,
+datadog.operator.go_memstats_mallocs_total,count,,,,TTotal number of mallocs,0,datadog_operator,go_memstats_mallocs_total,
+datadog.operator.go_memstats_mcache_inuse_bytes,gauge,,byte,,Number of bytes in use by mcache structures,0,datadog_operator,go_memstats_mcache_inuse_bytes,
+datadog.operator.go_memstats_mcache_sys_bytes,gauge,,byte,,Number of bytes used for mcache structures obtained from system,0,datadog_operator,go_memstats_mcache_sys_bytes,
+datadog.operator.go_memstats_mspan_inuse_bytes,gauge,,byte,,Number of bytes in use by mspan structures,0,datadog_operator,go_memstats_mspan_inuse_bytes,
+datadog.operator.go_memstats_mspan_sys_bytes,gauge,,,,Number of bytes used for mspan structures obtained from system,0,datadog_operator,go_memstats_mspan_sys_bytes,
+datadog.operator.go_memstats_next_gc_bytes,gauge,,byte,,Number of heap bytes when next garbage collection will take place,0,datadog_operator,go_memstats_next_gc_bytes,
+datadog.operator.go_memstats_other_sys_bytes,gauge,,byte,,Number of bytes used for other system allocations,0,datadog_operator,go_memstats_other_sys_bytes,
+datadog.operator.go_memstats_stack_inuse_bytes,gauge,,byte,,Number of bytes in use by the stack allocator,0,datadog_operator,go_memstats_stack_inuse_bytes,
+datadog.operator.go_memstats_stack_sys_bytes,gauge,,byte,,Number of bytes obtained from system for stack allocator,0,datadog_operator,go_memstats_stack_sys_bytes,
+datadog.operator.go_memstats_sys_bytes,gauge,,byte,,Number of bytes obtained from system,0,datadog_operator,go_memstats_stack_sys_bytes,
+datadog.operator.go_threads,gauge,,thread,,Number of OS threads created,0,datadog_operator,go_threads,
+datadog.operator.ksm.feature.enabled,gauge,,,,`1` if the Kubernetes State Metrics Core feature is enabled,0,datadog_operator,ksm_enabled,
+datadog.operator.live_container.feature.enabled,gauge,,,,`1` if the Live Containers feature is enabled,0,datadog_operator,live_containers_enabled,
+datadog.operator.log_collection.feature.enabled,gauge,,,,`1` if the Log Collection feature is enabled,0,datadog_operator,log_collection_enabled,
+datadog.operator.orchestrator_explorer.feature.enabled,gauge,,,,`1` if the Kubernetes Orchestrator Explorer feature is enabled,0,datadog_operator,orchestrator_explorer_enabled,
+datadog.operator.process_cpu_seconds_total,count,,,,Total user and system CPU time spent in seconds,0,datadog_operator,process_cpu_seconds_total,
+datadog.operator.process_max_fds,gauge,,,,Maximum number of open file descriptors,0,datadog_operator,process_max_fds,
+datadog.operator.process_open_fds,gauge,,,,Number of open file descriptors,0,datadog_operator,process_open_fds,
```

Comment:
we usually use the `file` unit for them IIRC

Reply:
Addressed by https://github.com/DataDog/integrations-core/pull/15655/commits/facfe66276ae175e6b0fb0244f78b2fb21d0677f